CodeStarterCamp_Week3/main.swift

```diff
@@ -1,12 +1,52 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week3
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
+class Person {
+    let name: String
+    var money: Int = 0
+    
+    init(name: String) {
+        self.name = name
+    }
+    
+    init(name: String, money: Int) {
+        self.name = name
+        self.money = money
+    }
```

Comment:
좋은방법인것 같습니다! 옛날옛적 자바를 배운 기억에 아래와 같이 수정해보았는데 맞는걸까요? 초기화할때 name만 들어가면 money의 기본값이 0이고, name과 money가 초기화로 들어가면 둘 다 초기화 잘되는거 확인은 했습니다. 그런데 스위프트도 이렇게 초기화 해도 되는건지 스위프트 문서를 찾아보니 보이지가 않습네요ㅠㅠ 혹시 키워드 힌트 부탁드려도 될까요?   
```
init(name: String, money: Int = 0) {
        self.name = name
        self.money = money
    }
```

Reply:
이니셜라이저 문서에는 제가 찾아보니 해당 내용이 없는 것 같고, `이니셜라이저도 함수다!` 라고 생각하면 편하실 것 같아요.
예를들어, print 함수의경우 `print(_:separator:terminator:)`로 정의되어있는데,
separator나 terminator의 경우 default value가 할당되어있기 때문에 
`print("Hello, World!")`만으로도 사용할 수 있는 것처럼, 
둘리가 만드신 이니셜라이저도 기본값이 할당되어있어서
name만 할당해주거나, 둘다 할당해줄 수 있는 형태로 사용할 수 있는거죠.

위 내용에 대한 것은 [Swift - Functions](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID159)에 Note로 나와있어요! 😊

> NOTE
The print(_:separator:terminator:) function doesn’t have a label for its first argument, and its other arguments are optional because they have a default value. These variations on function syntax are discussed below in [Function Argument Labels and Parameter Names](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID166) and [Default Parameter Values](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID169).

Reply:
아하~ 프린트 함수를 예로 설명해주셔서 이해가 바로갔습니다 설명 최고입니다!! 링크 보내주신 함수부분 한번 더 살펴보겠습니다 감사합니다!!